test/stylelintrc-css-modules/.stylelintrc.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
This is almost verbatim the config my team uses.  To demonstrate before/after, see `"property-no-unknown": true`; this fails on certain constructs without `stylelint-config-css-modules`.  In practice though, the missing dependency causes a different error...